sdks/python/apache_beam/testing/util.py

```diff
@@ -334,3 +335,19 @@ def open_shards(glob_pattern, mode='rt', encoding='utf-8'):
         out_file.write(in_file.read())
     concatenated_file_name = out_file.name
   return io.open(concatenated_file_name, mode, encoding=encoding)
+
+
+class TemporaryDirectory(object):
```

Comment:
Thanks. I forgot we are still on Py2, but very soon we can start removing Py2 support.

We actually defined this at least twice already. 

https://github.com/apache/beam/blob/0098eb6f9900849b05bfc2ec2497ed8c7d37148a/sdks/python/apache_beam/utils/subprocess_server_test.py#L39

https://github.com/apache/beam/blob/0098eb6f9900849b05bfc2ec2497ed8c7d37148a/sdks/python/apache_beam/testing/test_utils.py#L43

Feel free to reuse one of them and add a comment # TODO(BEAM-8371): Use tempfile.TemporaryDirectory.

Reply:
Thank you. I was not aware of them. I used this first one since it returns path on enter.